GovUkDesignSystem.SnapshotTests/GovUkDesignSystemComponents/Snapshots/CookieBannerTests.Render_AditionalCookies.approved.html

```diff
@@ -0,0 +1 @@
+﻿ 
```

Comment:
Don't need this file?

Reply:
Technically If the bannerState is hidden I want an empty file... right?